### ISSUE TITLE
ExpressionChangedAfterItHasBeenCheckedError in radioButton on reactive form

### ISSUE THREAD

**Issue by Prontsevich** [primefaces/primeng#2761]
**I'm submitting a ...**
```
[X ] bug report => Search github for a similar issue or PR before submitting
```

**Plunkr Case (Bug Reports)**
http://plnkr.co/edit/mgP4N2uobMTuUx5XQ1f8

**Current behavior**
Throw exception ExpressionChangedAfterItHasBeenCheckedError

**Expected behavior**
Radio buttons should work

**Minimal reproduction of the problem with instructions**
Look at Plunker demo.
On my page I have a group of radio buttons with colors. I show this group only after clicking on button 'Show more'. If I use reactive form and form control has initial value I get exception after showing radio buttons.

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **Angular version:** 4.1.0

* **PrimeNG version:** 4.0.0

* **Browser:** checked in Chrome, but can be in all

 
* **Language:** TypeScript 2.3.2 | ES5

**Comment by fkolar** [comment]
I am expiriencing the same problem.  Found this: https://github.com/angular/angular/issues/15464

The plunker does not work for me. 

Will try ngModel in meantime.

**Comment by Prontsevich** [comment]
ngModel works good in my example

**Comment by fkolar** [comment]
Yes, ngModel does work as a workaround for now.

**Comment by saithis** [comment]
We experience the same problem with angular 4.2.1 and primeng 4.0.3

**Comment by cagataycivici** [comment]
Duplicate of #2238

**Comment by cagataycivici** [comment]
Not a duplicate so reopen.
